.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: trusty
+sudo: required
```

Comment:
hmm. why we don't need this before?

Reply:
It looks like a regression in travis. I am not sure why we didn't need it before. I will spend some time figuring out why.

Reply:
I checked the travis docs. It looks like whenever you explicitly use `sudo make` or any `sudo <command>`, travis will assume `sudo: required` flag. This is mentioned here: https://docs.travis-ci.com/user/reference/overview/#For-a-particular-.travis.yml-configuration

When sudo is required, then build runs on GCE, otherwise in container on EC2. Here is the comparison:

- master branch (without any changes) [KCOV PASS]: https://travis-ci.org/sunchao/parquet-rs/builds/364441406, "Worker information" mentions GCE.
- build with `sudo: required` [KCOV PASS]: https://travis-ci.org/sunchao/parquet-rs/builds/365997814, "Worker information" also mentions GCE.
- build without any `sudo` keywords [KCOV FAIL]: https://travis-ci.org/sunchao/parquet-rs/builds/365537349, "Worker information" lists EC2.

So we have been running with `sudo: required` all this time, because of this line https://github.com/sadikovi/parquet-rs/blob/master/.travis.yml#L18.

So now we just explicitly state that we require sudo (for kcov issue on travis).

Reply:
I see. Nice finding! Thanks for the detailed explanation.